src/store/sagas/requests/adminQueryUsers.js

```diff
@@ -26,20 +26,21 @@ export const getAllTraders = async () => {
     let traders = []
     for (let userInfo of data.docs) {
         let user = await db.collection('users').doc(userInfo.data().uid).get()
-        traders.push({ ...user.data(), id: userInfo.data().uid })
+        traders.push({ ...user.data(), uid: userInfo.data().uid })
     }
     return traders
 }
 
-export const approveUser = async (data) => {
-    await db.doc(`users/${data.id}/private/info`).update({
+export const alterUser = async (data) => {
```

Comment:
This function shouldn't be an async function. We really don't need to worry if the `firstName`, `lastName` and `phone` is updated bcos the user can still do this himself if something wrong happens on the admin end.

But we are particular about the `private` collection being updated bcos that is the whole purpose of the update user process.... so we need the promise returned from updating the private collection --- we need to know if it failed or pass.

with this technique, we'll never know bcos you're not even returning the promise to the `handler`.

Reply:
The implementation from my last commit on this function is better aligned to this.

Reply:
```
export const alterUser = (data) => {

    db.doc(`users/${data.uid}`).update({
        firstName: data.firstName,
        lastName: data.lastName,
        phone: data.phone
    })
    
    return db.doc(`users/${data.uid}/private/info`).update({
        role: data.role,
        isApproved: true,
        ROI: data.roi < 0 ? 0 : data.roi,
        assignedTrader: data.role.isUser ? data.assignedTrader : "",
        affiliateCode: data.role.isAffliate ? data.affiliateCode : "",
        affiliate: data.affiliate
    })
}
```

Change the whole func to this...